CMakeLists.txt

```diff
@@ -89,13 +90,12 @@ if(WANT_OPENSSL)
 		# LibreSSL gained that function later too.
 		# TODO: maybe remove this check at some point, and stop supporting old
 		# libssl versions
-		function(check_SSL_SESSION_get0_cipher)
+		cmake_push_check_state(RESET)
 			set(CMAKE_REQUIRED_LIBRARIES ${OPENSSL_LIBRARIES})
```

Comment:
since this is not a function anymore, please fix indent.

Alternatively, maybe it makes sense to indent the one below instead